canvas.go

```diff
@@ -7,7 +7,11 @@ import "image"
 type Canvas interface {
 	Content() CanvasObject
 	SetContent(CanvasObject)
+	
 	Refresh(CanvasObject)
+	
+	// Focus makes the provided item focused
+	// Must be called after the item has been added to the contents of the canvas.
```

Comment:
The “must” looks like I have to call this method. What about:
```suggestion
	// The item has to be added to the contents of the canvas before calling this.
```